### DISCUSSION TITLE
THIS NEEDS TO BE FIXED ASAP!!

### DISCUSSION THREAD

**Discussion by SenTheNastyDynast** [neonbjb/tortoise-tts#253] [Ideas]
![image](https://user-images.githubusercontent.com/120703588/212576079-ffd1ff65-7002-42f6-8c0d-ef5f85fff0e3.png)

After lots of troubleshooting I was finally able to find out why this wasn't working on my 3090, this is literally setting a batch size of 16 which is bonkers, and completely unnecessary. I don't know nor want to know how to do a commit, I'm just letting you guys know, set the batch size on the api.py to 4 or lower and the thing will run without issues if you are having problems.

**Comment by neonbjb** [comment]
A batch size of 16 should easily fit in a 3090's memory. I own several 3090s and have explicitly tested this. I'm not sure what is wrong with your set-up.
